starknet_py/net/models/address.py

```diff
@@ -53,3 +54,4 @@ def compute_address(
 
 
 BlockIdentifier = as_our_module(BlockIdentifier)
+EthBlockIdentifier = as_our_module(EthBlockIdentifier)
```

Comment:
I think using web3 types is ok as it is a widely known library and it only used when communicating with L1.